_data/3.11-manual.yml

```diff
@@ -441,101 +328,11 @@
-          href: foxx-guides-cluster.html
-        - text: Scripts and scheduling
-          href: foxx-guides-scripts.html
-        - text: Using Node modules
-          href: foxx-guides-bundled-node-modules.html
-        - text: Using Webpack with Foxx
-          href: foxx-guides-webpack.html
-        - text: Authentication and sessions
-          href: foxx-guides-auth.html
-        - text: Linking services together
-          href: foxx-guides-dependencies.html
-        - text: Working with files
-          href: foxx-guides-files.html
-        - text: Making requests
-          href: foxx-guides-making-requests.html
-        - text: Access from the browser
-          href: foxx-guides-browser.html
-    - text: Reference
-      href: foxx-reference.html
-      children:
-        - text: Service manifest
-          href: foxx-reference-manifest.html
-        - text: Service context
-          href: foxx-reference-context.html
-        - text: Configuration
-          href: foxx-reference-configuration.html
-        - text: Routers
-          href: foxx-reference-routers.html
-          children:
-            - text: Endpoints
-              href: foxx-reference-routers-endpoints.html
-            - text: Middleware
-              href: foxx-reference-routers-middleware.html
-            - text: Request
-              href: foxx-reference-routers-request.html
-            - text: Response
-              href: foxx-reference-routers-response.html
-        - text: Sessions middleware
-          href: foxx-reference-sessions.html
-          children:
-            - text: Session storages
-              href: foxx-reference-sessions-storages.html
-              children:
-                - text: Collection storage
-                  href: foxx-reference-sessions-storages-collection.html
-                - text: JWT storage
-                  href: foxx-reference-sessions-storages-jwt.html
-            - text: Session transports
-              href: foxx-reference-sessions-transports.html
-              children:
-                - text: Cookie transport
-                  href: foxx-reference-sessions-transports-cookie.html
-                - text: Header transport
-                  href: foxx-reference-sessions-transports-header.html
-        - text: Related modules
-          href: foxx-reference-modules.html
-          children:
-            - text: Authentication
-              href: foxx-reference-modules-auth.html
-            - text: OAuth 1.0a
-              href: foxx-reference-modules-oauth1.html
-            - text: OAuth 2.0
-              href: foxx-reference-modules-oauth2.html
-            - text: GraphQL
-              href: foxx-reference-modules-graph-ql.html
-            - text: Queues
-              href: foxx-reference-modules-queues.html
-    - text: Deployment
-      href: foxx-deployment.html
 - text: SatelliteCollections
   href: satellites.html
 - text: SmartJoins
   href: smartjoins.html
```

Comment:
Yes, these topics that fit nowhere... Does it make sense to place them under Replication chapter? Or somewhere close to Cluster? If not, the only option I see is in Collections...

Reply:
They are features that you as a solution architect or developer need to plan for to take advantage of the performance benefits, so "modeling data" / "plan and design solutions" would be fitting. We could also consider adding a chapter about scalability features, but would we move the graph-related features to that place instead of having them under Graphs?